pkg/storage/system_intake_grb_reviewer.go

```diff
@@ -89,3 +89,30 @@ func (s *Store) CompareSystemIntakeGRBReviewers(ctx context.Context, systemIntak
 		"system_intake_id": systemIntakeID,
 	})
 }
+
+func (s *Store) UpdateSystemIntakeGRBReviewType(
+	ctx context.Context,
+	tx *sqlx.Tx,
```

Comment:
This is the other spot I was calling out, I think that this could be a named preparer.

Reply:
Let's centralize the conversation around `pkg/graph/resolvers/system_intake_grb_reviewer.go` and I'll reflect the resolution here